Mathlib/Logic/Basic.lean

```diff
@@ -1060,61 +1069,79 @@ theorem BEx.imp_left (H : ∀ x, p x → q x) : (∃ (x : _) (_ : p x), r x) →
   | ⟨x, hp, hr⟩ => ⟨x, H _ hp, hr⟩
 #align bex.imp_left BEx.imp_left
 
-theorem ball_of_forall (h : ∀ x, p x) (x) : p x := h x
-#align ball_of_forall ball_of_forall
+theorem forall_mem_of_forall (h : ∀ x, p x) (x) : p x := h x
+#align ball_of_forall forall_mem_of_forall
```

Comment:
Actually this is a stupid lemma. Can you revert the rename and deprecate it to `id`?